stable_baselines3/her/her.py

```diff
@@ -365,3 +369,146 @@ def __store_transitions(self) -> None:
 
                 # store data in replay buffer
                 self.replay_buffer.add(obs, new_obs, action, new_reward, done)
+
+    def __getattr__(self, item):
+        """
+        Find attribute from model class if this class does not have it.
+        """
+        if hasattr(self.model, item):
+            return getattr(self.model, item)
+        else:
+            raise AttributeError
+
+    def get_torch_variables(self) -> Tuple[List[str], List[str]]:
+        return self.model.get_torch_variables()
+
+    def save(
```

Comment:
is there no quicker way of doing that? (without duplicating too much code)

I would store HER specific arguments in the model (self.model.__dict__) , see what is done in SB2.